src/rules/data-quality/schedule-exceptdates-match-recurrence-dates.js

```diff
@@ -0,0 +1,69 @@
+const { RRule } = require('rrule');
+const Rule = require('../rule');
+const generateRRuleOptions = require('../../helpers/rrule-options');
+const ValidationErrorType = require('../../errors/validation-error-type');
+const ValidationErrorCategory = require('../../errors/validation-error-category');
+const ValidationErrorSeverity = require('../../errors/validation-error-severity');
+const getScheduleProperties = require('../../helpers/schedule-properties');
+
+module.exports = class ExceptDatesAreInSchedule extends Rule {
+  constructor(options) {
+    super(options);
+    this.targetModels = ['Schedule'];
+    this.meta = {
+      name: 'ExceptDatesAreInSchedule',
+      description:
+        'Validates that the Schedule contains exceptDates that are part of the recurrence rule.',
+      tests: {
+        exDate: {
+          message:
+            '{{value}} must be one of the events generated by the recurrence rule.',
+          sampleValues: {
+            startTime: '08:30',
+            endTime: '09:30',
+            startDate: '2021-03-19',
+            repeatFrequency: 'P1W',
+            count: 10,
+            exceptDates: ['2021-03-26T08:30:00Z'],
+            scheduleTimezone: 'Europe/London',
```

Comment:
`sampleValues` are used to generate https://validator.openactive.io/about, so should match the placeholder in the `message`. In this case only the key `value` needs to be supplied.

Reply:
Thanks, I don't think I really understand how or where this being used. I should have looked into that. I've updated this and the following related comments.